src/bun.js/modules/BunJSCModule.h

```diff
@@ -369,8 +368,19 @@ JSC_DEFINE_HOST_FUNCTION(functionCreateMemoryFootprint,
     size_t peak_commit = 0;
     size_t page_faults = 0;
 
+#if USE(MIMALLOC)
     mi_process_info(&elapsed_msecs, &user_msecs, &system_msecs, &current_rss,
         &peak_rss, &current_commit, &peak_commit, &page_faults);
+#else
+    (void)elapsed_msecs;
+    (void)user_msecs;
+    (void)system_msecs;
```

Comment:
Why not move the declarations on 362-364 within the `#if USE(MIMALLOC)` guard?

Reply:
they need to be accessible after the guard

Reply:
i see you meant just the ignored ones